### PR TITLE
Refactor workers

### PR DESCRIPTION
**Pull request by shimonewman**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,4 +1,4 @@
-_APP_ENV=development
+_APP_ENV=production
```
Comment (lohanidamodar) [inline .env:1]: Why set to production any particular reason?

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
Looking in great shape already. couple of suggestions to think of.

File: .env (hunk 1 above)
Comment (shimonewman) [reply .env:1]: No reason, tried the cert worker, but i guess I need a live deployment

File: app/workers/audits.php
[the remark this answers is not in the document]
Comment (shimonewman) [reply app/workers/audits.php:1]: Yes , waited for the last sec (:

**Pull request merged**